Wox.Plugin/Result.cs

```diff
@@ -14,6 +14,8 @@ public class Result
         public string Title { get; set; }
         public string SubTitle { get; set; }
 
+        public string ActionKeywordAssigned { get; set; }
```

Comment:
can we add a documentation here that will explain how it is going to be used and suggest keeping it in null in any case where you have a single action keyword plugin.

Reply:
Sure thing